modules/ignite-cli-2/pom.xml

```diff
@@ -126,6 +129,17 @@
             <version>${h2.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
+            <version>${assertj-core.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.github.stefanbirkner</groupId>
```

Comment:
This library appears to be unused, in case it will be used don't forget to use test scope